en/rules.md

```diff
@@ -157,7 +155,7 @@ If you want to participate in a score hunt, [Join our Discord](https://discord.g
 *make sure to read the requirements of __each__ scorehunt. they can change depending on any of them.*
 
 the following will **ALWAYS** be true for scorehunts.
```

Comment:
For each scorehunt, the following will **ALWAYS** be applied:

Reply:
157; it's worded this way to make sure anything can be included, that way relevant information about score hunts can just be in that one section. 

Changes for 145-55 pushed in  [fc6e68](https://github.com/lkse/website-docs/commit/fc6e685fb3717be2fc166601c48808333757e0f7)

Reply:
Yeah I guess is the same. Do what you want